packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -428,6 +428,18 @@ export class TaskDefinition extends TaskDefinitionBase {
   private findContainer(containerName: string): ContainerDefinition | undefined {
     return this.containers.find(c => c.containerName === containerName);
   }
+
+  private isEphemeralPortMappingSupported(networkMode: NetworkMode): boolean {
+    return networkMode === NetworkMode.BRIDGE || networkMode === NetworkMode.NAT;
+  }
+
+  private renderNetworkMode(networkMode: NetworkMode): string | undefined {
+    // 'NAT' is the only supported network mode for Windows containers,
```

Comment:
Since it is also explained in the ENUM def, I think you can combine those lines of comment into the ENUM one.

Reply:
Sure, makes sense. Thanks!